pkgs/applications/networking/ipfs/default.nix

```diff
@@ -1,26 +1,35 @@
-{ stdenv, buildGoPackage, fetchFromGitHub, fetchgx }:
+{ stdenv, buildGoPackage, fetchFromGitHub, fetchgx
+, makeWrapper, fuse
+}:
 
 buildGoPackage rec {
   name = "ipfs-${version}";
-  version = "0.4.10";
-  rev = "4679f806bd00c0a5299c22c82d1fbfdbad928e6d";
+  version = "0.4.11-pre";
+  rev = "v${version}";
 
   goPackagePath = "github.com/ipfs/go-ipfs";
 
   extraSrcPaths = [
     (fetchgx {
       inherit name src;
-      sha256 = "1khlsahv9vqx3h2smif5wdyb56jrza415hqid7883pqimfi66g3x";
+      sha256 = "0fvqnxr7scpfmnfny5xqsfcqs5m1phf1bd9ph2vs3ilvvzbprl88";
     })
   ];
 
   src = fetchFromGitHub {
     owner = "ipfs";
     repo = "go-ipfs";
     inherit rev;
-    sha256 = "1j3az0nhjisb5dxp1a4g8w17y17xjikvcsy4qrg0fm43ybpkhhvw";
+    sha256 = "03fvfr1ih0m12wpjvfavw67l03pnj3f7z33wjm8j9r4b700rc5qr";
   };
 
+  buildInputs = [ makeWrapper ];
+
+  postFixup = ''
+    wrapProgram $bin/bin/ipfs \
+      --prefix PATH : ${fuse}/bin
```

Comment:
Are you sure that this will work? The fuse tools need SUID root to work IIRC which isn't possible in the Nix store.

Reply:
That's true, but it's still better than "fuse not found" that will always be printed without this. I'll add a note in the autoMount option description that mentions the need to run the service as root for it to work.

Reply:
It is not a good idea to run IPFS as root just because of the SUID problem.
Wouldn't it be better to use `security.wrappers` instead?

-> nixos/modules/security/wrappers/default.nix

Reply:
Probably; I didn't know about those, thanks. I will try it out and update this PR today.